src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -203,13 +203,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     // this removes any HTML from the message content
     String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.none());
 
+    boolean privateMessage = false;
```

Comment:
.. or equivalently
boolean privateMessage = conversation.getPrivate();

(1 line instead of 4 :))

Reply:
oh yes that's a lot neater :) fixed!